Sources/Afterpay/Views/PaymentButton.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  PaymentButton.swift
+//  Afterpay
+//
+//  Created by Adam Campbell on 27/11/20.
+//  Copyright © 2020 Afterpay. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public final class PaymentButton: UIButton {
```

Comment:
How is voice-over accessibility handled with this class?

Reply:
Thanks for checking that, it'll need an accessibility label 👍